Plugins/NativeXr/Source/NativeXr.cpp

```diff
@@ -924,24 +953,43 @@ namespace Babylon
                 return info.Env().Global().Get(JS_CLASS_NAME).As<Napi::Function>().New({info[0]});
```

Comment:
Nit: Call your new overload?
```suggestion
                return New(info.Env(), info[0]);
```

Reply:
Good point!

Reply:
Actually on second look this would not cover the case where we pass in a napi::string, which is different from an object.